central/apitoken/service/service_impl_test.go

```diff
@@ -0,0 +1,53 @@
+package service
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stackrox/rox/central/role"
+	roleMock "github.com/stackrox/rox/central/role/datastore/mocks"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	"github.com/stackrox/rox/pkg/grpc/authn"
+	"github.com/stackrox/rox/pkg/grpc/authn/mocks"
+	"github.com/stackrox/rox/pkg/testutils/roletest"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/mock/gomock"
+)
+
+func TestServiceImpl_ListAllowedRoles_SortsRoleAlphabetically(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	mockIdentity := mocks.NewMockIdentity(mockCtrl)
```

Comment:
same here

Reply:
Ok, thank you for your comments, I have implemented the requests and the test is still passed  👍